src/main/java/edu/wpi/first/wpilibj/networktables/NetworkTableType.java

```diff
@@ -0,0 +1,40 @@
+package edu.wpi.first.wpilibj.networktables;
+
+/**
+ * Network table data types.
+ */
+public enum NetworkTableType {
+  kUnassigned(0),
+  kBoolean(0x01),
+  kDouble(0x02),
+  kString(0x04),
+  kRaw(0x08),
+  kBooleanArray(0x10),
+  kDoubleArray(0x20),
```

Comment:
And kNumber instead of kDouble?  Probably for consistency, although the C API uses the Double nomenclature instead of Number.  We could easily define both, I guess.